apps/modernization-ui/src/apps/search/investigation/GeneralSearchFields.tsx

```diff
@@ -100,13 +100,13 @@ const GeneralSearchFields = () => {
             <Controller
                 control={form.control}
                 name="pregnancyStatus"
-                render={({ field: { name, value } }) => (
+                render={({ field: { name } }) => (
                     <SingleSelect
                         data-testid="pregnancyStatus"
                         name={name}
                         id="pregnancyStatus"
                         label="Pregnancy test"
-                        onChange={() => form.setValue('pregnancyStatus', value)}
+                        onChange={(e) => form.setValue('pregnancyStatus', e)}
```

Comment:
Question: Can you pass the controller's `onChange` instead of calling `form.setValue` directly?

from `GeneralFields`
![image](https://github.com/user-attachments/assets/13a7ddbc-9c6d-4877-8463-e40b83c8f04b)

Reply:
thats how i was doing it originally, but Tony reported the value wasn't sticking on change

Reply:
If so, then that would indicate a bug in the `GeneralFields` as well.